apps/image-focal-point/package.json

```diff
@@ -36,5 +38,8 @@
     "> 1%",
     "not ie <= 11"
   ],
-  "homepage": "."
+  "homepage": ".",
+  "optionalDependencies": {
+    "@rollup/rollup-linux-x64-gnu": "4.9.5"
```

Comment:
Curious about this too since I've been having some weird rollup issues with optimizely and wondering if you were running into rollup stuff that this fixed?

Reply:
yeah I was, this fixed it :-)

Reply:
the error I was getting that this resolved was 
`Cannot find module @rollup/rollup-linux-x64-gnu`